api/all.yaml

```diff
@@ -1490,6 +1811,202 @@ components:
+          type: string
+        storage_location:
+          description: Storage root URL for model (for **MANAGED**, **EXTERNAL** tables)
+          type: string
+        comment:
+          description: User-provided free-form text description.
+          type: string
+        created_at:
+          description: Time at which this model was created, in epoch milliseconds.
+          type: integer
+          format: int64
+        updated_at:
+          description: Time at which this model was last modified, in epoch milliseconds.
+          type: integer
+          format: int64
+        model_id:
+          description: Unique identifier for the model.
+          type: string
+    ModelVersionInfo:
+      type: object
+      properties:
+        model_name:
+          description: Name of model, relative to parent schema.
+          type: string
+        catalog_name:
+          description: Name of parent catalog.
+          type: string
+        schema_name:
+          description: Name of parent schema relative to its parent catalog.
+          type: string
+        version:
+          description: The version number of the model version
+          type: integer
+          format: int64
+        source:
+          description: URI indicating the location of the source model artifacts.
+          type: string
+        run_id:
+          description: The run id used by the ML package that generated the model.
+          type: string
+        status:
+          $ref: '#/components/schemas/ModelVersionStatus'
+        storage_location:
+          description: Storage root URL for model version (for **MANAGED**, **EXTERNAL** tables)
+          type: string
+        comment:
+          description: User-provided free-form text description.
+          type: string
+        created_at:
+          description: Time at which this model version was created, in epoch milliseconds.
+          type: integer
+          format: int64
+        updated_at:
+          description: Time at which this model version was last modified, in epoch milliseconds.
+          type: integer
+          format: int64
+        model_version_id:
+          description: Unique identifier for the model version.
+          type: string
+    ModelVersionStatus:
+      description: All possible model version statuses
+      type: string
+      enum:
+        - UNKNOWN
+        - PENDING
+        - FAILED
+        - READY
+    CreateModel:
+      type: object
+      properties:
+        model_name:
+          description: Name of model, relative to parent schema.
+          type: string
+        catalog_name:
+          description: Name of parent catalog.
+          type: string
+        schema_name:
+          description: Name of parent schema relative to its parent catalog.
+          type: string
+        storage_location:
+          description: Storage root URL for table (for **MANAGED**, **EXTERNAL** tables)
```

Comment:
Same comment as above, why refer to tables here?

Reply:
Should be model...fixing